app/lib/provider/network/nearby_devices_provider.dart

```diff
@@ -138,7 +151,10 @@ class UnregisterSignalingDeviceAction extends ReduxAction<NearbyDevicesService,
 class StartMulticastScan extends ReduxAction<NearbyDevicesService, NearbyDevicesState> {
   @override
   NearbyDevicesState reduce() {
+    // broadcast to discover devices
     external(notifier._isolateController).dispatch(IsolateSendMulticastAnnouncementAction());
+    // log scan start
```

Comment:
Please remove the unnecessary comments like "log start"
Comments are supposed to explain the "why" of the code, not the "how" that can be understood from the code itself